grype/presenter/config.go

```diff
@@ -11,13 +11,14 @@ import (
 
 // Config is the presenter domain's configuration data structure.
 type Config struct {
-	format           format
-	templateFilePath string
+	format            format
+	templateFilePath  string
+	includeSuppressed bool
 }
 
 // ValidatedConfig returns a new, validated presenter.Config. If a valid Config cannot be created using the given input,
 // an error is returned.
-func ValidatedConfig(output, outputTemplateFile string) (Config, error) {
+func ValidatedConfig(inclSuppressed bool, output, outputTemplateFile string) (Config, error) {
```

Comment:
nit: I'd add this flag as the last parameter and write it out fully as `includeSuppressed`